9.2.74.md (#836)
modyo/modyo-docs: docs/en/platform/release-notes.md

# Release Notes

## 9.2
### 9.2.74 - April 02, 2025
<Badge text="Core" type="core" vertical="middle"/>

* Security enhancements
  
<Badge text="Channels" type="channels" vertical="middle"/>

> * Corrects problem of csp nonce differences between header and view

Reviewer: ```suggestion
* Fixes csp nonce differences problem between the header and the view
```
